docs/GettingStartedDocs/buildandsign.md

```diff
@@ -97,6 +97,15 @@ NumHeapPages=1024
 NumStackPages=1024
 NumTCS=1
 ```
+Additionally, for platform that supports for Key sharing and separation (KSS),
```

Comment:
```suggestion
Additionally, for platforms that support Key Sharing and Separation (KSS),
```

Reply:
How does one know whether KSS is supported?  Maybe add a link to which document says. E.g., is it reported by the oesgx tool?